test/Feature/RangeSpec.hs

```diff
@@ -199,62 +197,98 @@ spec = do
 
     context "when count=planned" $ do
       it "obtains a filtered range" $ do
-        request methodGet "/items?select=id&id=gt.8" [("Prefer", "count=planned")] ""
-          `shouldRespondWith` [json|[{"id":9}, {"id":10}, {"id":11}, {"id":12}, {"id":13}, {"id":14}, {"id":15}]|]
-          { matchStatus  = 206
-          , matchHeaders = ["Content-Range" <:> "0-6/8"]
-          }
-        request methodGet "/child_entities?select=id&id=gt.3" [("Prefer", "count=planned")] ""
-          `shouldRespondWith` [json|[{"id":4}, {"id":5}, {"id":6}]|]
-          { matchStatus  = 206
-          , matchHeaders = ["Content-Range" <:> "0-2/4"]
-          }
-        request methodGet "/getallprojects_view?select=id&id=lt.3" [("Prefer", "count=planned")] ""
-          `shouldRespondWith` [json|[{"id":1}, {"id":2}]|]
-          { matchStatus  = 206
-          , matchHeaders = ["Content-Range" <:> "0-1/673"]
-          }
+        request methodGet "/items?select=id&id=gt.8"
```

Comment:
Some tests are reformatted, because I touched them intermediately, but then reverted that and added data instead. So some tests technically are not changed but just look better ;)